examples/energy-management-app/esp32/main/Kconfig.projbuild

```diff
@@ -185,3 +185,50 @@ depends on ENABLE_PW_RPC
             about available pin numbers for UART.
 
 endmenu
+
+
+menu "Energy Management Example App Configuration"
+
+    config ENABLE_EXAMPLE_EVSE_DEVICE
+        bool "Enable EVSE device on EP1"
+        default "y"
+        help
+            To use Energy Management App as EVSE device and compile dependent files.
+
+    config ENABLE_EXAMPLE_WATER_HEATER_DEVICE
+        bool "Enable Water Heater device on EP1"
+        default "n"
+        help
+            To use Energy Management App as Water Heater device and compile dependent files.
+
+    config CHIP_DEVICE_CONFIG_ENABLE_DEVICE_ENERGY_MANAGEMENT_TRIGGER
+        bool "Enable DEM triggers"
+        default "y"
```

Comment:
Either this should be on by default and the other should be, but suspect for security we would want these OFF by default, and then in the SDKDefaults have them as set?

Reply:
Note Linux has all test event triggers enabled by default so maybe align this to the same?

Reply:
Code updated accordingly.